python/ml4ir/base/model/cyclic_learning_rate.py

```diff
@@ -0,0 +1,281 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
```

Comment:
@mohazahran You should rename this file to `lr_schedule` or something generic like that so that we can add more later on. I also think it might be a good idea to move this and `optimizer.py` into an `optimizers/...` directory.